src/NewFamilyMember.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from 'react';
+import './NewFamilyMember.scss';
+
+enum DocumentType {
+  type,
```

Comment:
I would recommend agains having an _'empty'_ enum value. Can you use `undefined` or `null` instead?